addons/pandora/plugin.gd

```diff
@@ -75,14 +76,15 @@ class PandoraExportPlugin extends EditorExportPlugin:
 	# Override the _export_begin method to add the data.pandora file during export
 	func _export_begin(features: PackedStringArray, is_debug: bool, path: String, flags: int):
 		var pandora_path = "res://data.pandora"
-		var file: FileAccess
-		if is_debug:
-			file = FileAccess.open(pandora_path, FileAccess.READ)
-		else:
-			file = FileAccess.open_compressed(pandora_path, FileAccess.READ)
-
-		if file:
-			add_file(pandora_path, file.get_buffer(file.get_length()), false)
+		var file = FileAccess.open(pandora_path, FileAccess.READ)
+		if not file:
+			printerr("Unable to export Pandora data: ",  FileAccess.get_open_error())
+			return
+		var data:PackedByteArray = file.get_buffer(file.get_length())
+		if not is_debug:
+			var text = file.get_as_text()
+			data = Compression.compress(text)
```

Comment:
This was the missing puzzle piece:

We need a way to take our uncompressed content (from the Godot game project) and store it in a compressed format, so `open_compressed` can understand it correctly.

Unfortunately, simply calling `compress()` on a `PackedByteArray` does not suffice, because Godot expects the following order of bytes:
```
magic
    char[4] "GCPF"

header
    uint32_t compression_mode (Compression::MODE_ZSTD by default)
    uint32_t block_size (4096 by default)
    uint32_t uncompressed_size

block compressed sizes, number of blocks = (uncompressed_size / block_size) + 1
    uint32_t block_sizes[]

followed by compressed block data, same as calling `compress` for each source `block_size`
```
Therefore, I implemented custom compression code, given that is not exposed to GDScript.